source/features/close-out-of-view-modals.tsx

```diff
@@ -1,29 +1,26 @@
 import onetime from 'onetime';
-import delegate from 'delegate-it';
 
 import features from '.';
 
 const observer = new IntersectionObserver(([{intersectionRatio, target}]) => {
 	if (intersectionRatio === 0) {
 		observer.unobserve(target);
-		target.closest('details')!.open = false;
+		target.closest('details')!.removeAttribute('open');
```

Comment:
I don't see a reason to change this / make it longer
```suggestion
 		target.closest('details')!.open = false;
```

The result is the same:

![a](https://user-images.githubusercontent.com/1402241/90086584-ab38d600-dd12-11ea-9c39-719ae34dc2c6.gif)